lib/stream.js

```diff
@@ -15,8 +15,15 @@ var log = isABib ? bunyan.createLogger({name: 'sierra-bib-retriever'}) : bunyan.
 // send data to kinesis Stream
 exports.streamPoster = (records, streamName) => {
   return writeToStreamsClient(streamName, records)
+          .then(() => {
+            return Promise.resolve({sent: records.length, receieved: records.length})
```

Comment:
is there a difference between sent and received? the value is the same since it uses records.length

CC: @jobinthomasnypl